hugo/content/3_how-tos/31_account-management/311_managing-projects.md

```diff
@@ -39,9 +39,14 @@ Managing projects include creating and deleting projects as well as adding and r
     1. Click **Create New**. 
        **Create new project** modal appears.
     2. In the **Name** field, enter the project name.
-    3. (Optional) Select the **Users** tab to add user to the project.
-    4. (Optional) Select the **User groups** tab to add user groups to the project.
-    5. Click **Create Project**. The newly created project can be viewed in the project dashboard.
+    3. To enable/disable VPN for the project, click the **Enabled/Disabled** button. For more information, see [Enabling VPN Services](/3_how-tos/34_networking-and-communication/347_enable-vpn/)
```

Comment:
Please check `Enabling VPN Services` link

Reply:
the link is working fine.